docs/WHOLE_STUDY_IMPORT.md

```diff
@@ -190,8 +188,23 @@ processed
 └── meta_sv.txt
 ```
 Note! Most other studies won't have a timeline set of files.
+
 ## Upload the final packages
-Upload all of the directories named as study short names to `s3://kf-strides-232196027141-cbioportal-studies/studies/`. You may need to set and/or copy your aws saml key before uploading. See "Access to [AWS Infra PedCBioPortal Import](https://github.com/d3b-center/aws-infra-pedcbioportal-import) repo" bullet point in [Software Prerequisites](../README.md#software-prerequisites) to load the study.
+Upload all of the directories named as study short names to `s3://kf-strides-232196027141-cbioportal-studies/studies/`. Authentication is done via **`aws sts assume-role`**. You can assume the appropriate role with the following command:
+```sh
+export $(printf "AWS_ACCESS_KEY_ID=%s AWS_SECRET_ACCESS_KEY=%s AWS_SESSION_TOKEN=%s" \
+$(aws sts assume-role \
+  --role-arn arn:aws:iam::232196027141:role/bix-dev-data-role \
+  --role-session-name MySessionName \
+  --query "Credentials.[AccessKeyId,SecretAccessKey,SessionToken]" \
+  --output text))
+```
+Once authenticated, upload your processed folder(s) using:
+```sh
+aws s3 cp processed/ s3://kf-strides-232196027141-cbioportal-studies/studies/ --recursive
```

Comment:
Oh, so this way you don't have to name the dir specifically, it'll just copy it?